Core/AppUrls.swift

```diff
@@ -172,5 +174,12 @@ public struct AppUrls {
         }
         return true
     }
+    
+    public func pixelUrl(forPixelNamed pixelName: String, deviceType: UIUserInterfaceIdiom = UIDevice.current.userInterfaceIdiom) -> URL {
+        let formFactor = deviceType == .pad ? "tablet" : "phone"
```

Comment:
Can we pull the strings into constants?

Optional: Can we move `let formFactor = deviceType == .pad ? "tablet" : "phone"` into `Pixel.swift`. It feels like it should provide the string it requires for a particular form factor.